Assets/MRTK/Core/Definitions/MixedRealityToolkitConfigurationProfile.cs

```diff
@@ -341,6 +341,19 @@ public bool RenderDepthBuffer
         [Tooltip("If true, MRTK will render the depth buffer as color. Only valid in editor.")]
         private bool renderDepthBuffer = false;
 
+        /// <summary>
+        /// If true, verbose logging will be enabled for MRTK components.
+        /// </summary>
+        public bool EnableVerboseLogging
+        {
+            get { return enableVerboseLogging; }
+            set { enableVerboseLogging = value; }
```

Comment:
Some of the files in our codebase use the more concise syntax:

```
get => enableVerboseLogging;
set => enableVerboseLogging = value;
```

Should this also follow this syntax too?

Reply:
Ah this is a part where I wanted to keep consistent with existing file (i.e. consistency with existing file, with folder, with project in that order).

I'd hold off on a single line prop change and want to update all of them at once.